src/styles/components/page-header/variables.less

```diff
@@ -34,71 +34,90 @@
 @page-header-sidebar-toggle-left-screen-jumbo: @page-header-sidebar-toggle-padding-screen-jumbo * -1;
 
 /*
- * Page Header Title
+ * Page Header Breadcrumbs
  */
 
 // Layout
 
-@page-header-title-padding-left: @icon-mini-width + @page-header-sidebar-toggle-padding + @page-header-sidebar-toggle-margin-right;
-@page-header-title-padding-left-screen-small: @icon-mini-width + @page-header-sidebar-toggle-padding-screen-small + @page-header-sidebar-toggle-margin-right-screen-small;
-@page-header-title-padding-left-screen-medium: @icon-mini-width + @page-header-sidebar-toggle-padding-screen-medium + @page-header-sidebar-toggle-margin-right-screen-medium;
-@page-header-title-padding-left-screen-large: @icon-mini-width + @page-header-sidebar-toggle-padding-screen-large + @page-header-sidebar-toggle-margin-right-screen-large;
-@page-header-title-padding-left-screen-jumbo: @icon-mini-width + @page-header-sidebar-toggle-padding-screen-jumbo + @page-header-sidebar-toggle-margin-right-screen-jumbo;
+@page-header-breadcrumbs-margin-right: @base-spacing-unit * 1/2;
+@page-header-breadcrumbs-margin-right-screen-small: @base-spacing-unit-screen-small * 1/2;
+@page-header-breadcrumbs-margin-right-screen-medium: @base-spacing-unit-screen-medium * 1/2;
+@page-header-breadcrumbs-margin-right-screen-large: @base-spacing-unit-screen-large * 1/2;
+@page-header-breadcrumbs-margin-right-screen-jumbo: @base-spacing-unit-screen-jumbo * 1/2;
+
+/*
+ * Page Header Content Sections
+ */
+
+// Layout
+
+@page-header-content-section-primary-padding-left: @icon-mini-width + @page-header-sidebar-toggle-padding + @page-header-sidebar-toggle-margin-right;
```

Comment:
Not blocking this PR but those variable assignments look funky and I wonder if we really need to calculate all of these paddings.